content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst

```diff
@@ -0,0 +1,320 @@
+  the product.
+- :guilabel:`Duration of Operations/Unit`: the average total duration of operations required to
+  produce one unit of the product.
+- :guilabel:`Quantity Demanded`: the total number of units of the product included in |MOs|.
+- :guilabel:`Quantity Produced`: the total number of units of the product that have actually been
+  produced.
+- :guilabel:`Total Component Cost`: the total amount spent on the product's components, across every
+  MO for the product.
+- :guilabel:`Total Cost`: the total amount spent manufacturing every unit of the product produced so
+  far.
+- :guilabel:`Total Duration of Operations`: the cumulative duration of every operation completed
+  while manufacturing the product.
+- :guilabel:`Total Employee Cost`: the cumulative amount paid to employees to manufacture the
+  product.
+- :guilabel:`Total Operation Cost`: the cumulative amount spent on operations required to produce
+  the product.
+- :guilabel:`Total Operation Cost/Unit`: the average cost of the operations required to produce
+  one unit of the product.
+- :guilabel:`Total Subcontracting Cost`: the cumulative amount paid to subcontractors to produce the
+  product.
+- :guilabel:`Yield Percentage (%)`: the total quantity of the product produced versus the total
+  quantity demanded, represented as a percentage.
+- :guilabel:`Count`: the total count of |MOs| created for the product.
+
+.. note::
+   Only one measure can be selected at a time.
+
+Views
+=====
+
+By default, the production analysis dashboard displays data using the *line chart* graph type of the
+*graph view*.
+
+To select the bar chart or pie chart graph type instead, click the :icon:`fa-bar-chart`
+:guilabel:`(bar chart)`, or :icon:`fa-pie-chart` :guilabel:`(pie chart)` button below the
+:guilabel:`Search...` bar. To return to the line chart, click the :icon:`fa-line-chart`
+:guilabel:`line chart` button.
+
+To see production data displayed in a pivot table instead of a graph, click the
+:icon:`oi-view-pivot` :guilabel:`(pivot view)`, button to the right of the :guilabel:`Search...`
+bar. To return to graph view, click the :icon:`fa-area-chart` :guilabel:`(graph view)` button.
+
+Bar chart
+---------
+
+The x-axis of the bar chart lists each week of the selected :guilabel:`End Date` time period. The
+y-axis displays ascending numbers representing the count of whichever metric was selected on the
+:guilabel:`Measure` drop-down menu (ex. Average Total Cost/Unit). Thus, each bar represents the
+total count during a single week of the selected time period.
+
+Each bar is divided into sections representing one of the products manufactured during that week. By
+default, the sections are stacked on top of one another to form a single bar.
+
+To view the sections as individual bars, click the :icon:`fa-database` :guilabel:`(stacked)` button
+to the right of the graph view buttons, to deactivate the stacking feature. Click the button again
+to return to the stacked view.
```

Comment:
I'm wondering if 'stacked view' should be capitalized...or hit with italics... 🤔 it might just be good as-is. But, it's something to think about during your revisions. 👍